cmd/oci-image-tool/man/oci-image-tool-unpack.1.md

```diff
@@ -8,8 +8,7 @@ oci-image-tool-unpack \- Unpack an image or image source layout
 **oci-image-tool unpack** [src] [dest] [flags]
```

Comment:
To reduce churn with this PR, it might be better to just use `dest` for the new stuff.  We can file a follow-up PR to handle placeholder CAPS.